components/menu/menu-item-styles.js

```diff
@@ -3,27 +3,26 @@ import { css } from 'lit-element/lit-element.js';
 
 export const menuItemStyles = css`
 	:host {
-		background-color: #ffffff;
+		background-color: var(--d2l-menu-background-color);
 		border: 1px solid transparent;
-		border-top-color: var(--d2l-color-gypsum);
+		border-top-color: var(--d2l-menu-border-color);
 		box-sizing: border-box;
-		color: var(--d2l-color-ferrite);
+		color: var(--d2l-menu-foreground-color);
```

Comment:
Do you want to make a similar change to line #6 in `dropdown-content-styles.js`?  It think it's the color that would be used for generic dropdown content (not a menu).  It looks like some of the other style changes apply to that scenario, so it seems _partly_ supported in the current state.

Reply:
So currently only `<d2l-dropdown-menu>` supports `theme="dark"` (it's the only one with the mixin)... so technically I think we can get away without changing `dropdown-content-styles.js`?

Reply:
Actually no you're right... since `dropdown-content-styles.js` is already using some of these variables, they NEED to be defined with it otherwise dropdowns that aren't menus won't work -- which is why a bunch of visual diffs failed.

Reply:
Ok fixed!